2022-01-18 10:20:18,976 INFO    MainThread:4952 [wandb_setup.py:_flush():71] setting env: {}
2022-01-18 10:20:18,977 INFO    MainThread:4952 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-18 10:20:18,977 INFO    MainThread:4952 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d01_pre_processing\wandb\run-20220118_102018-2rg1z1c0\logs\debug.log
2022-01-18 10:20:18,978 INFO    MainThread:4952 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d01_pre_processing\wandb\run-20220118_102018-2rg1z1c0\logs\debug-internal.log
2022-01-18 10:20:18,978 INFO    MainThread:4952 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002661164A640>
2022-01-18 10:20:18,989 INFO    MainThread:4952 [wandb_init.py:init():404] calling init triggers
2022-01-18 10:20:18,989 INFO    MainThread:4952 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-18 10:20:18,990 INFO    MainThread:4952 [wandb_init.py:init():449] starting backend
2022-01-18 10:20:18,990 INFO    MainThread:4952 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2022-01-18 10:20:19,000 INFO    MainThread:4952 [backend.py:ensure_launched():199] starting backend process...
2022-01-18 10:20:19,319 INFO    MainThread:4952 [backend.py:ensure_launched():204] started backend process with pid: 15808
2022-01-18 10:20:19,320 INFO    MainThread:4952 [wandb_init.py:init():458] backend started and connected
2022-01-18 10:20:19,326 INFO    MainThread:4952 [wandb_run.py:_label_probe_notebook():917] probe notebook
2022-01-18 10:20:19,327 INFO    MainThread:4952 [wandb_run.py:_label_probe_notebook():927] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-01-18 10:20:19,327 INFO    MainThread:4952 [wandb_init.py:init():519] updated telemetry
2022-01-18 10:20:19,334 INFO    MainThread:4952 [wandb_init.py:init():549] communicating current version
2022-01-18 10:20:20,692 INFO    MainThread:4952 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-18 10:20:20,692 INFO    MainThread:4952 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-01-18 10:20:20,911 INFO    MainThread:4952 [wandb_init.py:init():592] starting run threads in backend
2022-01-18 10:20:21,871 INFO    MainThread:4952 [wandb_run.py:_console_start():1816] atexit reg
2022-01-18 10:20:21,872 INFO    MainThread:4952 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2022-01-18 10:20:21,873 INFO    MainThread:4952 [wandb_run.py:_redirect():1727] Wrapping output streams.
2022-01-18 10:20:21,874 INFO    MainThread:4952 [wandb_run.py:_redirect():1751] Redirects installed.
2022-01-18 10:20:21,874 INFO    MainThread:4952 [wandb_init.py:init():619] run started, returning control to user process
2022-01-18 10:20:23,046 INFO    MainThread:4952 [wandb_run.py:finish():1463] finishing run austinbergstrom/places_dry_run/2rg1z1c0
2022-01-18 10:20:23,156 INFO    MainThread:4952 [jupyter.py:save_history():429] not saving jupyter history
2022-01-18 10:20:23,156 INFO    MainThread:4952 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-18 10:20:23,156 INFO    MainThread:4952 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-01-18 10:20:23,156 INFO    MainThread:4952 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 0
2022-01-18 10:20:23,157 INFO    MainThread:4952 [wandb_run.py:_restore():1758] restore
2022-01-18 10:20:23,395 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 649
}

2022-01-18 10:20:24,027 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 649
}

2022-01-18 10:20:24,529 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 649
}

2022-01-18 10:20:24,923 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 4314
}

2022-01-18 10:20:25,025 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 4340
}

2022-01-18 10:20:25,130 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 649
  total_bytes: 4340
}

2022-01-18 10:20:25,232 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1008
  total_bytes: 4340
}

2022-01-18 10:20:25,337 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1034
  total_bytes: 4340
}

2022-01-18 10:20:25,445 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1034
  total_bytes: 4340
}

2022-01-18 10:20:25,546 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1034
  total_bytes: 4340
}

2022-01-18 10:20:25,652 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4340
  total_bytes: 4340
}

2022-01-18 10:20:25,756 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4340
  total_bytes: 4340
}

2022-01-18 10:20:25,859 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4340
  total_bytes: 4340
}

2022-01-18 10:20:25,964 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4340
  total_bytes: 4340
}

2022-01-18 10:20:26,113 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4340
  total_bytes: 4340
}

2022-01-18 10:20:26,609 INFO    MainThread:4952 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4340
  total_bytes: 4340
}
local_info {
}

2022-01-18 10:20:27,758 INFO    MainThread:4952 [wandb_run.py:_append_files():2184] logging synced files
